### DISCUSSION TITLE
Downgrading to GitHub Free

### DISCUSSION THREAD

**Discussion by marchbold** [community/community#23291] [New to GitHub]
<p>I’m trying to understand what will happen when we move our organisation to GitHub Free. The latest plans seem to suit us perfectly except for a small few migration questions I cant find the answers to.</p>
<p>We currently are on a legacy gold plan and using private repositories. We have detailed usage instructions of our code in the wikis associated with these private repository.</p>
<p>What will happen to these wikis when we move to GitHub free as they seem to not be included for private repositories on the free plan? </p>
<p>Also, is there an unlimited number of organisation members allowed, i,e the same as collaborators on the free plan?</p>

**Comment by lesleyanneb** [comment]
<p>Hi marchbold,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the GitHub Account Questions board.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/marchbold">@marchbold</a>,</p>
<p>Not to worry! Your wikis will only be disabled while you are on the Free tier after moving from your legacy plan.  If you choose to re-enable wikis later by upgrading to GitHub <a href="https://github.com/pricing" title="Pricing" rel="nofollow noopener">Team</a>, your previous pages will be restored.</p>
<p>While organizational members are unlimited with Free, your features within the Free tier for private repositories will be limited for organizations. We’d recommend checking out what’s <a href="https://help.github.com/en/github/getting-started-with-github/githubs-products#github-free-for-organizations" title="GitHub Products, Free Organizations" rel="nofollow noopener">included</a> on our GitHub Products page.</p>
<p>I hope this helps!</p>

**Comment by shulsher** [comment]
<p>Thanks for the help :smiley:</p>
